rules/falco_rules.yaml

```diff
@@ -2599,3 +2599,9 @@
 # there if you want to enable them by adding to
 # falco_rules.local.yaml.
 
+- rule: The docker client is executed in a container
+  desc: Detect a k8s client tool executed inside a container
+  condition: spawned_process and container and proc.name in (k8s_client_binaries)
```

Comment:
Did you accidentally delete the `k8s_client_binaries` list? And please move the rule above 
```
# Application rules have moved to application_rules.yaml. Please look
# there if you want to enable them by adding to
# falco_rules.local.yaml.
```

Reply:
Thanks again Kaizhe. You are really being patience with me in this PR...

Reply:
Thank you for your contribution! It is really appreciated!